builders/tools/walletconnect.md

```diff
@@ -0,0 +1,416 @@
+---
+title: Integrate WalletConnect
+description: Learn how to integrate WalletConnect into a dApp built on any of the Moonbeam networks
+---
+
+# Integrate WalletConnect into a DApp
+
+![WalletConnect Banner](/images/builders/tools/walletconnect/walletconnect-banner.png)
+
+## Introduction {: #introduction } 
+
+[WalletConnect](https://walletconnect.com/) is an open protocol to communicate securely between wallets and DApps. 
+
+WalletConnect establishes a remote connection between a DApp and mobile wallet by using a bridge server to relay payloads. The connection is initiated via a QR code displayed in a DApp, which will need to be scanned and approved by a mobile wallet. When a connection is established, the payloads between the DApp and wallet are encrypted through a shared key.
+
+![WalletConnect flow](/images/builders/tools/walletconnect/walletconnect-1.png)
```

Comment:
I like the idea of the image but I feel is a bit incomplete. This might help on providing more details https://dojo.coz.io/images/walletconnect-diagram.png

Reply:
yeah I saw a variation of that image but was trying to keep it more simple

Reply:
do you think I should add all of the pieces from the image in that link? or do you think there are some parts that can be left out for simplicity?

Reply:
I would add the least amount of pieces but while still providing the necessary context (easy to write, not that easy to do haha)

Reply:
haha well I thought it had enough context before so I'm not really sure what to add and what to leave out 😂